cmd/serviceaccnt.go

```diff
@@ -0,0 +1,66 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	serivceAccount "github.com/method-security/methodk8s/internal/serviceaccount"
```

Comment:
Typo in the alias. `serviceAccount`.

Why do we need the alias in the first place here though?